### ISSUE TITLE
Standardize the subject lines of messages between tools admins and tools users

### ISSUE THREAD

**Issue by annlarson** [debtcollective/parent#201]
It is very hard to follow a thread of messages in Discourse when the subject line keeps changing. Admins are talking to multiple users at once and users themselves may have filed more than one dispute. Admins are requesting that each message contain a standard subject heading with information about that identifies the user, the dispute ID, and the status of the dispute. That way, every time a message is posted it will be clear what is being discussed. 

Our suggestion is that EVERY message regarding a dispute tool contain a subject line with the following template (or something similar) :

Admin Update for: Tim Jones- ID023- Status:ContactAdmin 

"Admin Update" means that an admin is responding to a users' dispute.
Tim Jones is the name of the user
ID023 is the unique dispute ID, corresponding to ONLY a single dispute
"Status: Contact Admin" is the status of the dispute. 

Standardized subject headings will help us keep track of all the messages that admins send to tools users and that users send back to admins.  Thanks,

**Comment by orlando** [comment]
@annlarson to clarify, with the changes we are making, we will have just **one private message** per Dispute, instead of creating a new one for every message an admin sends. In our last call we mentioned this template.

> <Dispute ID> - <Full name> - <Dispute type>

With the data above will look like this.

> 023 - Tim Jones - Tax Offset Dispute

We don't have the status in the title since that may change multiple times, but the ID, name and dispute type won't. 

What do you think?

**Comment by annlarson** [comment]
Sounds good! So if there are multiple messages and replies it will be in one message? Thanks.

**Comment by orlando** [comment]
Yes, only one private message that can have multiple replies

Enviado desde mi iPhone

**Comment by annlarson** [comment]
This is great. Thanks!

**Comment by orlando** [comment]
@annlarson I'm merging this ticket with #194.

Closing
